packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1453,14 +1355,9 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   /// Subclasses can override this getter to make its value depend on an inherited
   /// theme.
   ///
-  /// Defaults to false when [RawScrollbar.isAlwaysShown] or
-  /// [RawScrollbar.thumbVisibility] is null.
```

Comment:
Do we want to keep "Defaults to false when `[RawScrollbar.thumbVisibility]` is null."?

Reply:
Ah my bad, I should've looked more carefully.